test/factories/user.js

```diff
@@ -0,0 +1,14 @@
+let factory = require('factory-girl');
```

Comment:
why let instead of const? you could just assign `.factory` directly to the variable instead of doing it below.

Reply:
That is because if not I cannot execute `adapter = new factory.SequelizeAdapter();` and since I need to change the factory to `factory.factory` I need it to declare it as let